gatsby/static/images/ss.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```

Comment:
This file can probably be deleted

Reply:
yeah @aaronraimist I just added it mistakenly